validator/client/src/test/java/tech/pegasys/teku/validator/client/ValidatorRegistratorTest.java

```diff
@@ -169,6 +169,23 @@ void registersValidators_shouldRegisterWithTimestampOverride() {
     verify(signer, times(1)).signValidatorRegistration(any());
   }
 
+  @TestTemplate
+  void registersValidators_shouldRegisterWithPubkeyOverride() {
+    when(validatorConfig.getBuilderRegistrationPubkeyOverride())
+        .thenReturn(Optional.of(BLSPublicKey.valueOf(140)));
```

Comment:
BLSPublicKey.valueOf(140) 

again need random BLSPublicKey not sure whats the best function to use or if a hardcoded value should be used